docs/appendices/release-notes/unreleased.rst

```diff
@@ -194,4 +194,6 @@ Changes
 Fixes
 =====
 
-None
+- Fixed an issue where updates of the tables metadata while doing
+  ``CREATE SNAPSHOT`` is not captured in the snapshot, resulting in potential
+  incompatibility problems when the table is restored.
```

Comment:
Suggestion:
Fixed an resiliency issue on snapshot creation while dynamic columns are created concurrently which may result in incompatibility problems on restore.